### PR TITLE
🔧 remove nuxt-property-decorator

### PR DESCRIPTION
**Pull request by roiLeo**
240 results in 233 files

### Context

- related with #4750 
- `vue/multi-word-component-names` rule (for like Avatar component name)
- <s>`/_prefix/collection/_id` & `/_prefix/u/_id`</s> done in #4966
- Vue 3 composition API
- remove westend & westmint for now
- Nuxt 3 auto imported components

### CODE REVIEW

**Comment by vikiival** [comment]
That is huge 😅

**Comment by vikiival** [comment]
> @roiLeo maybe create a separate PR for pages/collection/_id.vue?

Yes please! 
I Would love to do that first

File: pages/carbonless.vue [hunk 1]
```diff
@@ -149,24 +149,3 @@
     </p>
   </section>
 </template>
-
-<script lang="ts">
-import { Component, Vue } from 'nuxt-property-decorator'
-
-@Component<Carbonless>({
-  head() {
-    const title = 'KodaDot cares about environmental impact'
```
[the remark this answers is not in the document]
Comment (roiLeo) [reply pages/carbonless.vue:158]: sort of now heriting parent meta

**Comment by yangwao** [comment]
time to break things? 🫠

**Comment by yangwao** [comment]
let's see

image

**Comment by yangwao** [comment]
> Looks like deploy is failing?

image

**Pull request merged**